linear_search/linear_search.js

```diff
@@ -0,0 +1,29 @@
+function linearSearch (arr, x) {
+	/*
+	Performs a linear search
+	:param arr: List of elements to search from
+	:param x: Element to search for
+	:return: Index if element found else -1
+	*/
+	for (let it in arr) {
+		if (arr[it] === x) {
+			return it;
+		}
+	}
+	return -1;
+}
+
+function main () {
+	let A = [];
+	for (let i = 1; i<=100; i++) {
```

Comment:
You are looking at the old commit, this is the latest https://github.com/iiitv/algos/pull/303/commits/80b47a7dcb6c77574e971d26473440fcd3f254e1